MapboxSearch/autofill/src/main/java/com/mapbox/search/autofill/AddressAutofillSuggestion.kt

```diff
@@ -23,13 +24,16 @@ public class AddressAutofillSuggestion internal constructor(
     /**
      * Address geographic point.
      */
-    public val coordinate: Point,
+    public val coordinate: Point?,
```

Comment:
In case of `AddressAutofill`, we'll never have coordinates at suggestions step. So, `coordinate` should be removed instead of making it non-null.

Reply:
Done